mail_debrand/tests/test_mail_debrand.py

```diff
@@ -2,18 +2,42 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from odoo.tests import common
+from odoo.tools.misc import mute_logger
 
 
 class TestMailDebrand(common.TransactionCase):
     def setUp(self):
         super().setUp()
+        self.other_lang = "it_IT"
+        self.env["res.lang"].load_lang(self.other_lang)
```

Comment:
Loading a language is quite expensive and here it is done again and again for each test because we are in `TransactionCase` subclass, could you please avoid reloading the language for each test?

You might switch to `SavepointCase` or just load the language for the test that needs it if it's just one.